skills/csharp/experimental/itsmskill/Deployment/Resources/LU/en-us/secondary/CreateSubscription.lu

```diff
@@ -0,0 +1,8 @@
+﻿# CreateSubscription
+
+> create
+
+- create a new subscription
+- add new subscription
+- add subscription
+- create subscription
```

Comment:
Wondering if these should be patterns (e.g. `[create|add] subscription`), because there may be confusion with create/add ticket?)